DEV.md

```diff
@@ -14,7 +14,7 @@ $ jq --version
 jq-1.6
 $ docker --version
 Docker version 20.10.7, build f0df350
-$ docker compose version
+$ docker-compose --version
```

Comment:
The original command works fine for me:
![image](https://user-images.githubusercontent.com/17320781/149737851-bf9efcbb-c564-4a63-b2ad-116df0ea6b00.png)

Reply:
Doesn't Work For Me.. Thats Strange
<img width="559" alt="Screenshot 2022-01-17 at 10 40 34" src="https://user-images.githubusercontent.com/37830553/149745611-f15fbafa-2800-4b20-9351-3aac4167059f.png">

Reply:
Are you running docker-compose v2?

Reply:
Needs docker-compose v2, this was a deliberate change we made some months back when we upgraded docker-compose requirements from v1 to v2 (see PR #779)

Reply:
I'm Still On v1, Will Upgrade To v2